webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.html

```diff
@@ -68,7 +68,7 @@
     </div>
     <div class="policy-violation-label flex flex-col" *ngIf="violationData.violationCreatedDate">
         <div class="policy-violation-label-header capitalize">
-            Creation date
+            Date Created 2/28/2023
```

Comment:
The column name should not contain a hardcoded date value. Please remove 2/28/2023.

The actual value for this is coming from line 74